.travis.yml

```diff
@@ -8,7 +8,7 @@ install:
 script:
   - rubocop -fs -D
   - bin/executable-tests-check
-  - rake test
+  - rake test:all
   - make test
```

Comment:
Can we make `rake test` an alias for `test:all`
This would mean we wouldn't need to change line 11 either.